kubernetes/adapter.go

```diff
@@ -35,6 +35,16 @@ var (
 	// ErrInvalidCertificates is returned when the CA certificates required to communicate with the
 	// API server are invalid
 	ErrInvalidCertificates = errors.New("invalid CA certificates")
+	// validSSLPolicy is a map of valid ALB SSL Policies
```

Comment:
Can you add the AWS link to the valid certs here?

Reply:
Added